src/main/java/Event.java

```diff
@@ -0,0 +1,18 @@
+public class Event extends Task {
```

Comment:
Perhaps consider having the suffix -Task added to classes that inherit from Task for clarity? E.g EventTask